packages/core/src/factory.ts

```diff
@@ -5,38 +5,23 @@ import * as TaskEither from 'fp-ts/lib/TaskEither';
 import { configMergerFactory, PartialPrismConfig, PrismConfig } from '.';
 import { IPrism, IPrismComponents, IPrismConfig, IPrismDiagnostic, PickRequired, ProblemJsonError } from './types';
 
-export function factory<Resource, Input, Output, Config, LoadOpts>(
+export function factory<Resource, Input, Output, Config>(
```

Comment:
Since the loader is not a thing anymore in Prism, we have one less template argument to take care of.